thriftpy/thrift.py

```diff
@@ -297,13 +293,23 @@ def process(self, iprot, oprot):
         if isinstance(result, TApplicationException):
             return self.send_exception(oprot, api, result, seqid)
 
+        has_exc = False
         try:
-            result.success = call()
+            success = call()
         except Exception as e:
             # raise if api don't have throws
             self.handle_exception(e, result)
+            has_exc = True
```

Comment:
It makes sense that  one-way calls don't even need to return an exception. But what's the behavior in official implementation?